packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -1376,6 +1380,513 @@ describe('Table', () => {
                 );
             });
         });
+
+        describe('delay loaded hierarchical data', () => {
+            const hierarchicalData: SimpleTableRecord[] = [
```

Comment:
nit: Since this initial data is flat, not hierarchical, maybe call it `initialData` or `parentData` or something like that?

Could also consider marking it [readonly](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#a-new-syntax-for-readonlyarray) or `as const` if the intent is that no test should modify its contents.